UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Storage/ExchangeSetStorageProvider.cs

```diff
@@ -10,9 +10,9 @@ namespace UKHO.ExchangeSetService.Common.Storage
     public class ExchangeSetStorageProvider : IExchangeSetStorageProvider
     {
         private readonly IOptions<EssFulfilmentStorageConfiguration> storageConfig;
-        private readonly IAzureBlobStorageClient azureBlobStorageClient;
+        private readonly IAzureBlobStorageService azureBlobStorageClient;
         public ExchangeSetStorageProvider(IOptions<EssFulfilmentStorageConfiguration> storageConfig,
-            IAzureBlobStorageClient azureBlobStorageClient)
+            IAzureBlobStorageService azureBlobStorageClient)
         {
```

Comment:
This method has been updated to use the IAzureBlobStorageService but the field is still named as if it is the Client.

Would it make sense to update this field to reflect the new Interface?

Reply:
refactored as split into method and removed unused properties

Reply:
renamed